.buildkite/steps/release-prepare.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+version="$(cat version/VERSION)"
+
+echo "--- :label: Checking git tag"
+
+# If the tag is already exists, we assume the release is for nightly build,
+# otherwise we assume it's a stable release
+if git ls-remote --tags origin | grep "refs/tags/v${version}" ; then
+  echo "v${version} already exists, running nightly release"
+  buildkite-agent meta-data set "release-phase" "nightly"
+  buildkite-agent meta-data set "version" "$version-$(git rev-parse --short HEAD)"
```

Comment:
@wooly we cannot trigger build in `Test Splitter Client - Release` pipeline from the `Test Splitter Client` pipeline, therefore we can not get the build number. I decided to tag it with the short commit sha instead. What is your thought? We can use the build number of the `Test Splitter Client - Release` pipeline, but seems a bit off to me.